internal/commands/platform/export_internal_test.go

```diff
@@ -73,7 +73,7 @@ func TestInitPingFederateServices(t *testing.T) {
 	}
 
 	// check pf context has auth values included
-	if pingfederateContext.Value(pingfederateGoClient.ContextOAuth2) == nil {
+	if pingfederateContext.Value(pingfederateGoClient.ContextBasicAuth) == nil {
 		t.Errorf("initPingFederateServices() pingfederateContext.Value = %v, want non-nil", pingfederateContext.Value(pingfederateGoClient.ContextOAuth2))
```

Comment:
Looks like this line should have `ContextOAuth2` switched to `ContextBasicAuth`.

Reply:
Fixed 👍